web/impact/impact/tests/test_snake_and_camel_casing.py

```diff
@@ -0,0 +1,16 @@
+from impact.tests.api_test_case import APITestCase
+from impact.models.utils import (
+    camel_case_to_snake,
+    snake_to_camel_case,
+)
+
+camel_case = 'ProgramStartupStatus'
+snake_case = 'program_startup_status'
+
+
+class TestSnakeAndCamelCasing(APITestCase):
```

Comment:
Our standards require that the name of this class (and module) should be `TestUtils` and `test_utils.py`, respectively (or the original functions should be moved).

Reply:
I don't understand this point.  Can you expound, please?

Reply:
Done!